src/Controls/tests/TestCases.HostApp/Issues/Issue18628.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Maui.Controls.Shapes;
+
+namespace Maui.Controls.Sample.Issues
+{
+
+	[Issue(IssueTracker.Github, 18628, "[Android/iOS] Rectangle rendering is broken")]
+	public class Issue18628 : ContentPage
+	{
+		public Issue18628()
+		{
+			var grid = new Grid
+			{
+				WidthRequest = 600,
+				HeightRequest = 100
+			};
+
+			var rectangle = new Rectangle
```

Comment:
The change should impact any Shape, could you include in the test other common shapes like an Ellipse for example?

Reply:
I have modified the test case based on your suggestion. Please review and share if any suggestions